tox.ini

```diff
@@ -6,30 +6,15 @@ max-line-length = 100
 
 [tox]
 isolated_build = true
-envlist = py,black,flake8,isort,mypy
+envlist = mypy,pre-commit,py
```

Comment:
What if we added the `Unrecognized environment name` functionality here as well? I am noticing that something like `tox -e black` succeeds, but of course it just runs the unit tests.

Reply:
Sure!